Tools/OSD/osd_screen.cs

```diff
@@ -604,12 +606,6 @@ private void rbtSortAlphabetic_CheckedChanged(object sender, EventArgs e)
 			}
 		}
 		
-        private int correct_NTSC(int y){
-            if (y >= osd.getCenter() && osd.is_ntsc()) 
-                y+=3;
-            return y;
-        }
-
```

Comment:
BTW, it's not about checking. There are 2 different things, one is the drag&drop and other is the up/down control. Unfortunatelly I couldn't reuse the same logic for both situations.